lib/package/base/server-request/api/zuri_api.dart

```diff
@@ -322,7 +325,6 @@ class ZuriApi implements Api {
     }
   }
 
-  @override
```

Comment:
Please why are you removing the @override?

Reply:
Its not in the interface so there's nothing for it to override